website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -64,37 +64,38 @@ Properties without a default value are required.
+| `data.attributes.assessments-enabled`           | boolean | `false`   | (previously `drift-detection`) Whether or not Terraform Cloud performs health assessments for the workspace. May be overriden by the organization setting `assessments-enforced`. Only available for Plus tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).                                                                                                                               |
+| `data.attributes.auto-apply`                    | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by VCS, UI or CLI, [with some exceptions](/terraform/cloud-docs/workspaces/settings#auto-apply-and-manual-apply).                                                                                                                                                                                                                                                                                                  |
+| `data.attributes.auto-apply-run-trigger`        | boolean | `false`   | Whether to automatically apply changes when a Terraform plan is successful in runs initiated by run triggers. Not supported in Terraform Enterprise.                                                                                                                                                                                                                                                                                                   |
+| `data.attributes.auto-destroy-at`               | string  | (nothing) | Timestamp when the next scheduled destroy run will occur, refer to [Scheduled Destroy](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy).                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.auto-destroy-activity-duration`| string  | (nothing) | Value and units for automatically scheduled destroy runs based on workspace activity. Values must be greater than 0 and valid units are `d` and `h`.                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
I think an example value would be super helpful for illuminating what the format is for this. 

Also I think a link would be helpful here:
```suggestion
| `data.attributes.auto-destroy-activity-duration`| string  | (nothing) | Value and units for [automatically scheduled destroy runs based on workspace activity](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy). Values must be greater than 0 and valid units are `d` and `h`.                                                                                                                                                                                                                                                                                                                                                            |
```

Reply:
Added this as an example:

```md
For example, to set to queue destroy runs after fourteen days of inactivity set `auto-destroy-activity-duration: "14d"`.
```